lms/djangoapps/courseware/index.py

```diff
@@ -0,0 +1,554 @@
+"""
+View for Courseware Index
+"""
```

Comment:
Standard django practice would be to put all views in views.py or (once there are too many) under `lms/djangoapps/courseware/views/`. Move views.py to views/*.py, and Optionally, keep urls.py cleaner (and calling code backwards compatible) by reexporting everything from `lms/djangoapps/courseware/views/__init__.py`.